app/assets/stylesheets/views/_homepage.scss

```
@@ -139,13 +135,13 @@ body.homepage {
 }
 
 .home-divider {
-  margin: 30px 0 15px 0;
+  margin: 30px 0 15px;
```

You could use `govuk-spacing` here for those margin values and for the other instances on line 148 to keep things consistent. [Details](https://design-system.service.gov.uk/styles/spacing/#spacing-on-custom-components). I think this would look like...
```suggestion
  margin: govuk-spacing(6) 0 govuk-spacing(3);
```

It's more verbose but protects us from any future (though maybe unlikely) spacing changes.

---

Works for me, updated 👍🏼